### PR TITLE
Run R cmd check for current release and devel version

### PR DESCRIPTION
**Pull request by shun2wang**
to check compiling error early for upcoming R version.

so jaspBase will fail to compile on Windows with R4.4.0+ . but this PR will not resolve it.

### CODE REVIEW

**Review by Kucharssim — APPROVED** [review]
This looks fine to me, should we wait for the merge of https://github.com/jasp-stats/jaspColumnEncoder/pull/8 before we merge this?

**Comment by shun2wang** [comment]
I think this is a separate chore change and you don't have to wait for it:-)

**Review by JorisGoosen — APPROVED** [review]

**Comment by JorisGoosen** [comment]
Im just going to move jaspColumnEncoder into jaspBase and then reuse it from there

**Pull request merged**
